enterprise_gateway/services/processproxies/container.py

```diff
@@ -200,12 +201,18 @@ async def confirm_remote_startup(self) -> None:
 
             container_status = self.get_container_status(str(i))
             if container_status:
-                if self.assigned_host != "":
-                    ready_to_connect = await self.receive_connection_info()
-                    self.pid = (
-                        0  # We won't send process signals for kubernetes lifecycle management
+                self.log.debug(f"Received startup status: {container_status}")
+                if container_status in self.get_error_states():
+                    self.log_and_raise(
+                        http_status_code=500, reason="Error starting kernel container."
```

Comment:
Could we add the status to this message?
```suggestion
                        http_status_code=500, reason=f"Error starting kernel container; status: '{container_status}'."
```